pygame_menu/widgets/core/selection.py

```diff
@@ -88,6 +89,19 @@ def get_width(self):
         _, l, _, r = self.get_margin()
         return l + r
 
+    def inflate(self, rect):
+        """
+        Grow or shrink the rectangle size according to margins.
+
+        :param rect: rectangle
+        :type rect: :py:class:`pygame.Rect`
+        """
+        assert isinstance(rect, pygame.Rect)
+        return pygame.Rect(rect.x - self.margin_left,
+                           rect.y - self.margin_top,
+                           rect.width + self.margin_left + self.margin_right,
+                           rect.height + self.margin_top + self.margin_bottom)
+
```

Comment:
This method should be used in HighlightSelection too?

Reply:
Yes, that's right